routes/users.js

```diff
@@ -1,55 +1,59 @@
-const Joi = require("joi");
-const app = require("express");
+const app = require('express');
+
+const _ = require('lodash')
 const router = app.Router();
-const User = require("../models/user");
+const User = require('../models/user');
 const {
   UserBodySchema,
-  UserBodyLoginSchema
-} = require("./validators/validators");
-const validator = require("express-joi-validator");
+} = require('./validators/validators');
+const validator = require('express-joi-validator');
+
+const bcrypt = require('bcrypt');
+
+const saltRounds = 10;
 
-/* GET users listing. */
-router.get("/", function(req, res, next) {
+router.get('/', (req, res, next) => {
   User.find({})
-    .then(records => {
+    .then((records) => {
       res.send(records);
     })
     .catch(next);
-  //return res.sendFile(path.join(__dirname + '/templateLogReg/index.html'));
 });
 
-router.post("/", validator(UserBodySchema), (req, res, next) => {
-  console.log(JSON.stringify(req.body));
+router.post('/', validator(UserBodySchema), (req, res, next) => {
   User.create(req.body)
-    .then(added => {
-      res.send({ message: "User sucesfully created!" });
-      //res.redirect("/login");
+    .then((added) => {
```

Comment:
I would use `user` here instead of `added`. I know it's more comfortable for copy-pasting but for readability it's better to use more self descriptive names.